src/_Problems_/next-greater-element/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+  You must implement the nextGreaterElement() function.
+  For each element in an array, it finds the next greater element in that array.
```

Comment:
![image](https://user-images.githubusercontent.com/50075905/65828378-df9cd280-e2b7-11e9-8670-63c5ec9f41d0.png)

Actually what you are doing is you are finding nextGreaterElement from i to (n-1) .
For eg: 
- For element  6 nextGreaterElement should be 8, not 10.